components/bl-backdrop-component/dist/index.js

```diff
@@ -0,0 +1 @@
'use strict';var e={ d:(t, l) => {for(const n in l)e.o(l, n)&&!e.o(t, n)&&Object.defineProperty(t, n, { enumerable:!0, get:l[n] })}, o:(e, t) => Object.prototype.hasOwnProperty.call(e, t), r:e => {'undefined'!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e, Symbol.toStringTag, { value:'Module' }), Object.defineProperty(e, '__esModule', { value:!0 })} }, t={};e.r(t), e.d(t, { default:() => c });const l=self.React, n={ 'line circle':function(){return React.createElement('div', { className:'bl-customComponent-lineCircleLoader' }, React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null))}, 'doted circle':function(){return React.createElement('div', { className:'bl-customComponent-dotedCircleLoader' }, React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null))}, bar:function(){return React.createElement('div', { className:'bl-customComponent-barLoader' }, React.createElement('div', null), React.createElement('div', null), React.createElement('div', null))}, 'doted line circle':function(){return React.createElement('div', { className:'bl-customComponent-dotedLineCircleLoader' }, React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null))}, radar:function(){return React.createElement('div', { className:'bl-customComponent-radarLoader' }, React.createElement('div', null), React.createElement('div', null))}, 'doted square':function(){return React.createElement('div', { className:'bl-customComponent-squareDotedLoader' }, React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null))}, 'doted horizont':function(){return React.createElement('div', { className:'bl-customComponent-dotedHorizontLoader' }, React.createElement('div', null), React.createElement('div', null), React.createElement('div', null), React.createElement('div', null))} };function c({ component:e, eventHandlers:t }){const{ backdropVisibility:c, loaderType:a, display:r }=e, { onClick:o, onOpen:d, onClose:i }=t;if((0, l.useEffect)((() => (d(), i)), []), !c&&!r)return null;const u=n[a];return React.createElement('div', { onClick:o, className:'bl-customComponent-backdrop' }, React.createElement(u, null))}return t})()))
```

Comment:
add `/dist` to gitignore

.gitignore must be located in `/component/<component-name>` directory